ui/raidboss/data/06-ew/trial/asura.ts

```diff
@@ -234,20 +234,21 @@ const triggerSet: TriggerSet<Data> = {
     },
     {
       id: 'Asura Iconic Execution With Jump',
-      type: 'CombatantMemory',
-      // Filter to only enemy actors for performance
-      netRegex: { id: '4[0-9A-Fa-f]{7}', capture: true },
+      type: 'ActorSetPos',
+      // The Asura Image doesn't have associated ActorSetPos lines except for this mechanic.
+      // Can't meaningfully filter regex as nearly all 271 lines are NPCs; but it's <100 in total.
+      netRegex: {},
       condition: (data, matches) =>
         data.iconicExecutionCount >= 3 &&
         data.asuraImageId === matches.id &&
         data.storedIconMech !== undefined,
       alertText: (data, matches, output) => {
-        if (matches.pairPosX === undefined || matches.pairPosY === undefined)
+        if (matches.x === undefined || matches.y === undefined)
           return;
```

Comment:
This condition shouldn't be needed anymore I believe. It was required previously because not every `CombatantMemory` line would have `PosX`/`PosY`.

Reply:
Huh, interesting.  I left it in because TypeScript almost always complains about undefined checks, but yeah, the type for that line doesn't allow undefined.

Reply:
For clarification, the check for undef on log lines is only for lines with optional matches, which is currently only `CombatantMemory` lines.

Basically the typedefs for log lines state "the matches object will always have these properties", followed up by "the matches object might have these properties".

If you're ever doing an undef check on a non-combatant-memory line it's probably a bug.